zuul.d/ansible-cloud-jobs.yaml

```diff
@@ -438,7 +438,7 @@
       ansible_test_collections: true
       ansible_test_command: integration
       ansible_test_python: 3.8
-      ansible_test_integration_targets: ""
+      ansible_test_integration_targets: zuul/aws/cloud_control
```

Comment:
Add a / at the end: `zuul/aws/cloud_control/`

Reply:
Still nothing.